lib/apiservers/portlayer/restapi/configure_port_layer.go

```diff
@@ -103,7 +103,7 @@ func configureAPI(api *operations.PortLayerAPI) http.Handler {
 // The middleware configuration is for the handler executors. These do not apply to the swagger.json document.
 // The middleware executes after routing but before authentication, binding and validation
 func setupMiddlewares(handler http.Handler) http.Handler {
-	return handler
+	return curler.New(handler, os.Stdout)
```

Comment:
ah that was left over from when I was debugging the swagger issue. I was going to ask @sflxn if we should just leave it so that logs in production have some curl output as well. I guess not. it is a debug thing.